book/src/user/docker.md

```diff
@@ -0,0 +1,38 @@
+# Zebra with Docker
+
+We've embraced Docker in Zebra for most of the solution lifecycle, from development environments to CI (in our pipelines), and deployment to end users. Zebra can be built from source using Cargo, Rust’s build system and package manager alternatively, you can easily deploy Zebra using [Docker](https://docs.docker.com/get-docker/)
+
+## Quick usage
+
+You can deploy Zebra for a daily use with the images available in [Docker Hub](https://hub.docker.com/repository/docker/zfnd/zebra) or build it locally for testing
+
+### Ready to use image
+
+```shell
+docker run -d zfnd/zebra:1.0.0-rc.0
+```
```

Comment:
The name of this file will need to be added to the release checklist, so we update the version when it changes:
https://github.com/ZcashFoundation/zebra/blob/161bb80ce8fc131bd7fea3a95723b34f1795716e/.github/PULL_REQUEST_TEMPLATE/release-checklist.md?plain=1#L45
https://github.com/ZcashFoundation/zebra/blob/161bb80ce8fc131bd7fea3a95723b34f1795716e/.github/PULL_REQUEST_TEMPLATE/release-checklist.md?plain=1#L62

Reply:
Fixed by: [docs: update checklist with docker user documentation](https://github.com/ZcashFoundation/zebra/pull/5504/commits/2bcc57ddc0951225b5d11545d59f5c52616c0d7d)